_data/authors.yml

```diff
@@ -42,6 +42,9 @@ lian-yuanlin:
   name: Lian Yuanlin
   thumbnail: /img/authors/lian-yuanlin.jpg
   github: alienchow
+nishant-gupta:
+  name: Nishant Gupta
+  thumbnail: /img/authors/nishant-gupta.jpg
```

Comment:
How big is the photo? Generally I try to resize to 256 x 256 size and compress first.